core/variables.js

```diff
@@ -192,6 +199,123 @@ const flyoutCategoryBlocks = function(workspace) {
 };
 exports.flyoutCategoryBlocks = flyoutCategoryBlocks;
 
+/**
+ * Generate DOM objects representing a variable field.
+ * @param {!VariableModel} variableModel The variable model to
+ *     represent.
+ * @return {?Element} The generated DOM.
+ * @deprecated
```

Comment:
Here especially I think specifying what should be used instead might help you answer the question you pose in your PR comment.